ui-ngx/src/app/modules/home/components/widget/config/basic/indicator/signal-strength-basic-config.component.html

```diff
@@ -109,6 +109,13 @@
                       formControlName="inactiveBarsColor">
       </tb-color-input>
     </div>
+    <div class="tb-form-row space-between">
+      <div translate>widgets.signal-strength.no-signal-rssi-value</div>
+      <mat-form-field appearance="outline" class="number" subscriptSizing="dynamic">
+        <input matInput formControlName="noSignalRssiValue" type="number" step="1"
+               placeholder="{{ 'widget-config.set' | translate }}">
+      </mat-form-field>
+    </div>
```

Comment:
Perhaps this setting does not need to be added to the basic form.

Reply:
Advanced form is indeed a better place for this setting